extensions/catalog/federated-catalog-cache/src/test/java/org/eclipse/dataspaceconnector/catalog/cache/TestProtocolAdapterRegistry.java

```diff
@@ -11,30 +11,32 @@
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 
-public class TestProtocolAdapterRegistry implements CatalogQueryAdapterRegistry {
-    private final Map<String, List<CatalogQueryAdapter>> map;
+public class TestProtocolAdapterRegistry implements NodeQueryAdapterRegistry {
```

Comment:
It's the same as `NodeQueryAdapterRegistryImpl` except for line 19, that should executed externally the constructor by the test. So my proposal is to remove this class.

Reply:
removed it.